eng/_util/cmd/run-builder/systemfips_linux.go

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+package main
+
+import (
+	"log"
+	"os"
+)
+
+// enableSystemWideFIPS enables Mariner and Azure Linux 3 process-wide FIPS mode.
```

Comment:
I think in context what's here is plenty, but I figure a bit more explicit detail doesn't hurt:

```suggestion
// enableSystemWideFIPS enables Mariner and Azure Linux 3 process-wide FIPS mode
// for any process that inherits the current process' environment variables.
```